.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+<!--- Are there any related issues? If so, please put the link here. -->
+<!--- Why is this change required? What problem does it solve? -->
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
```

Comment:
Could we make this section simpler?

1. Tell user the command to run the tests `test_all_pipeline.py` and  just ask if user has run it? 
2. How to run users new test (if appropriate)

Reply:
Ok, let me see how to tell users run the test script.